src/main/java/com/klarna/hiverunner/sql/cli/hive/PreV130HiveCliEmulator.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2015-2018 Klarna AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.klarna.hiverunner.sql.cli.hive;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.klarna.hiverunner.sql.HiveSqlStatementFactory;
+import com.klarna.hiverunner.sql.cli.CommandShellEmulator;
+import com.klarna.hiverunner.sql.cli.PostProcessor;
+import com.klarna.hiverunner.sql.cli.PreProcessor;
+import com.klarna.hiverunner.sql.split.CloseStatementRule;
+import com.klarna.hiverunner.sql.split.DefaultTokenRule;
+import com.klarna.hiverunner.sql.split.PreserveCommentsRule;
+import com.klarna.hiverunner.sql.split.PreserveQuotesRule;
+import com.klarna.hiverunner.sql.split.StatementSplitter;
+import com.klarna.hiverunner.sql.split.TokenRule;
+
+/**
+ * Emulates CLI behaviours specific to the Hive CLI. This includes
```

Comment:
Prior to Hive version "a.b.c". Is this Hive 0.13? I don't think there is an official 1.3.0 release of it? It went from Hive 1.2.2 to some 2.x release AFAIK.